[COPS-758] Error descriptions for secrets manager (#520)
neo4j/docs-status-codes: modules/ROOT/pages/errors/gql-errors/51N7C.adoc

= 51N7C

== Status description
error: system configuration or operation exception - secret provider is not enabled. Secret provider `{ <<secretProviderType>> }` is not available. Enable it with the setting `{ <<cfgSetting>> }`.

== Explanation

> An attempt was made to call a secret provider that has not been configured.

Reviewer: We try to use the active voice and the present tense. Maybe something like this:
```suggestion
This error occurs when you call a secret provider that has not been configured.
```

> The secret provider must be enabled in the system configuration file before calling any methods on it. 

Reviewer: What do you mean by "system configuration"? Is this the neo4j.conf file or any other conf file?

Author: Yeah neo4j.conf

> The secret provider must be enabled in the system configuration file before calling any methods on it. 

Reviewer: ```suggestion
The secret provider must be enabled in the _neo4j.conf_ file before calling any methods on it. 
```
